docs/howtos/consuming-rust-components-on-ios.md

```diff
@@ -14,7 +14,8 @@ The package is published as a release on github: https://github.com/mozilla/appl
 - Add additional dependencies, see [below](#additional-dependencies).
 
 ### Adding a carthage provided framework to Xcode
-- In general, to do this, add *XXX.framework* from *Carthage/Build/iOS* to *Link binary with Libraries* for the Xcode target, and also to the *Copy Carthage Dependencies* step
+- In general, to do this, add *XXX.framework* from *Carthage/Build/iOS* to *Link binary with Libraries* for the Xcode target
+- If you are using the megazord version of the Rust component, `MozillaAppServices.framework`, it is being built as a static lib; therefore, do _not_ follow the standard Carthage procedure of adding it to your `carthage copy-frameworks` script.
```

Comment:
```suggestion
- `MozillaAppServices.framework` is being built as a static lib; therefore, do _not_ follow the standard Carthage procedure of adding it to your `carthage copy-frameworks` script.
```

Since we don't ship single components anymore